lib/wptool.js

```diff
@@ -832,8 +832,13 @@ function wpToolInstall(deployCmd, device, appPath, options, callback) {
 		if (code) {
 			// handle duplicate package identity error code from Windows 10.0.14393 tooling and above
 			if (code == '2148734208') {
-				options.forceUnInstall = true;
-				wpToolInstall(deployCmd, device, appPath, options, callback);
+				if (out.indexOf('because the current user does not have that package installed') == -1) {
+					options.forceUnInstall = true;
+					wpToolInstall(deployCmd, device, appPath, options, callback);
+				} else {
+					// Windows cannot remove the app because the current user does not have that package installed.
+					callback(new Error('A debug application is already installed, please remove existing debug application'));
+				}
```

Comment:
Note: How this works:

1. When different app is installed with same product id, the command fails to install new one
2. It returns the code `2148734208`, and then we try to uninstall it (`wpToolInstall`)
3. Then the command fails to uninstall because it has different package name
4. It returns the code 2148734208 again with message `current user does not have that package installed`
5. Then we end up displaying error message